src/lib.rs

```diff
@@ -264,7 +264,7 @@ impl WapcHost {
     /// It is worth noting that the _first_ time `call` is invoked, the WebAssembly module
     /// will be JIT-compiled. This can take up to a few seconds on debug .wasm files, but
     /// all subsequent calls will be "hot" and run at near-native speeds.    
-    pub fn call(&mut self, op: &str, payload: &[u8]) -> Result<Vec<u8>> {
+    pub fn call(self, op: &str, payload: &[u8]) -> Result<Vec<u8>> {
```

Comment:
Could the first argument be `&self` instead of `self`? That would be most suitable for https://github.com/wascc/wascc-host/pull/44

Reply:
Crap, that's actually I a typo. It absolutely should be `&self`